projects/Mallard/src/authentication/lib/Attempt.ts

```diff
@@ -64,6 +71,17 @@ const ValidAttemptCons = <T>(
     time,
 })
 
+const ErrorAttemptCons = <T>(
```

Comment:
What does `Cons` mean here?

Reply:
Constructor, but `ErrorAttempt` is also a type so there's a naming collision. We only export the constructor though so it's renamed on `ErrorAttempt` on export.